documentation/Ingress-controller-upgrade.md

```diff
@@ -0,0 +1,36 @@
+# Ingress Controller Upgrade
+
+**Pre-requisits**
+- AKS Cluster with the ingress controller you wish to upgrade
+- An app deployed to the cluster to monitor i.e. apply, register etc
```

Comment:
The welcome apps can be used as they are always deployed: https://test.teacherservices.cloud/, https://platform-test.teacherservices.cloud/ etc